test/fixtures/url-tests.js

```diff
@@ -2,7 +2,7 @@
 
 /* The following tests are copied from WPT. Modifications to them should be
    upstreamed first. Refs:
-   https://github.com/w3c/web-platform-tests/blob/88b75886e/url/urltestdata.json
+   https://github.com/web-platform-tests/wpt/blob/ba4921d054808a44c2966507c19861ca729a76a2/url/resources/urltestdata.json
```

Comment:
Minor nit: perhaps we could make this URL at least a little shorter by reducing the length of the commit/tree hash?